spec/models/course_spec.rb

```diff
@@ -769,4 +769,134 @@
       end
     end
   end
+
+  describe '#copy_to_provider' do
```

Comment:
would be nice to have tests that covers

`new_course` 
`content_status` should be either `draft` or `empty` covers the enrichments
`ucas_status` should be `new` or `not running` (if original is also not running) covers the site statuses
`open_for_applications` should be nil

my thinking is along the lines that a rollovered course will have the state as listed above as those exact states will be presented to user on frontend